index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+   
+    <div class="flex-container">
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo">My Logo</li>
```

Comment:
- [ ] I assume this a kind of name logo. I suggest using a-tag in addition to the li-tag to make it a link.